extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js

```diff
@@ -7,24 +7,36 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements', [
 	'use strict';
 
 	var logGroup = 'ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements',
-		placementsMap = instantGlobals.wgAdDriverAppNexusBidderPlacementsConfig;
+		placementsMap = instantGlobals.wgAdDriverAppNexusBidderPlacementsConfig,
+		palPlacementsMap = {
+			oasis: {
+				atf: '3143028',
+				btf: '3143029',
+				hivi: '3142398',
+				other: '3140173'
+			}
```

Comment:
I'm wondering if we could go either one way or the other, that is move all placement ids to the code (like is for other providers) OR keep them all in one place (wf right now)

Reply:
per conversation with @nefretete we decided to move all AN placements into code :) Don't care about wg variable set on dev env.